interp/src/interpreter/interpret_control.rs

```diff
@@ -44,12 +44,42 @@ fn eval_seq(
 /// Interpret Par
 
 fn eval_par(
-    _p: &ir::Par,
-    _continuous_assignments: &[ir::Assignment],
-    mut _env: InterpreterState,
-    _comp: &ir::Component,
+    p: &ir::Par,
+    continuous_assignments: &[ir::Assignment],
+    mut env: InterpreterState,
+    comp: &ir::Component,
 ) -> FutilResult<InterpreterState> {
-    todo!("par control operator")
+    let mut sts = Vec::new();
+
+    for st in &p.stmts {
+        sts.push(interpret_control(
+            st,
+            continuous_assignments,
+            //change fork to do fork first and then tail
+            env.fork(),
+            comp,
+        )?);
+    }
+
+    sts.push(env);
+
+    let mut tl = 0;
+    let mut sms = Vec::new();
```

Comment:
a more descriptive variable name would be helpful as `sts` and `sms` parse similar